core/templates/services/user.service.ts

```diff
@@ -61,6 +61,8 @@ export class UserService {
   }
 
   getProfileImageDataUrl(username: string): [string, string] {
+    // TODO(17663): Remove use of performance.now with a long term fix
```

Comment:
```suggestion
    // TODO(#17663): Remove use of performance.now with a long term fix
```

Reply:
Done, submitted the testing request as well.